numpy/core/_add_newdocs.py

```diff
@@ -795,7 +795,10 @@
     ----------
     object : array_like
         An array, any object exposing the array interface, an object whose
-        __array__ method returns an array, or any (nested) sequence.
+        __array__ method returns an array, or any (nested) sequence. However
+        if a number is passed as a parameter then it will return the number itself.
```

Comment:
Hi @yashasvimisra2798, the lint test is failing because this line has more than 79 characters.